code/main/edge/buildEdgeRequestURL.brs

```diff
@@ -13,12 +13,20 @@
 
 ' ***************************** MODULE: buildEdgeRequestURL *******************************
 
-function _adb_buildEdgeRequestURL(configId as string, requestId as string, edgeDomain = invalid as dynamic) as string
+function _adb_buildEdgeRequestURL(configId as string, requestId as string, path as string, edgeDomain = invalid as dynamic) as string
     scheme = "https://"
     host = "edge.adobedc.net"
-    path = "/ee/v1/interact"
     query = "?configId=" + configId
 
+    if not _adb_isEmptyOrInvalidString(path) then
```

Comment:
nit: I would also prefer removing these checks and make sure we pass the path in correct format from our code.